src/pandas_profiling/report/presentation/flavours/html/templates/alerts.html

```diff
@@ -4,23 +4,22 @@
         {% if alerts is mapping %}
             <thead>
                 <tr>
-                    {% for index in range(style.labels | length) %}
-                        <th style="color: {{ style.primary_colors[index] }};">{{ style.labels[index] }}</th>
+                    {% for index in range(style._labels | length) %}
+                        <th style="color: {{ style.primary_colors[index] }};">{{ style._labels[index] }}</th>
```

Comment:
It feels icky to have to access an `_internal` field here. Maybe just give `labels` an accessor property instead of all of these changes?

Reply:
yeah, it seems like labels was a public field and was later refactored into an internal one since this configuration is more of an auxiliary field than a proper configuration. Here I just completed the refactoring that missed some of the changes.

Reply:
Agree with @akx it would be great to NOT have private fields in HTML template.